src/main/webapp/js/constants/field-configs.js

```diff
@@ -0,0 +1,523 @@
+import {
+  ROLE_ENTITY,
+  PERMISSION_ENTITY,
+  LANGUAGE_ENTITY,
+  CAMPAIGN_MESSAGE_ENTITY,
+  VISIT_TYPE_ENTITY,
+  USER_ENTITY,
+} from './entity-types';
+
+import {
+  TEXT,
+  COLLECTION,
+  INTEGER,
+  RELATION,
+  ENUM,
+  BOOLEAN,
+  DATE_TIME,
+} from './field-types';
+
+export const roleFieldConfig = [{
+  base: true,
+  editable: false,
+  filterable: true,
+  required: false,
+  hidden: true,
+  fieldType: TEXT,
+  entity: ROLE_ENTITY,
+  displayName: 'ID',
+  name: 'id',
+  fieldOrder: 0,
+  relatedEntity: null,
+  relatedField: null,
+  format: '',
+}, {
+  base: true,
+  editable: true,
+  filterable: true,
+  required: true,
+  hidden: false,
+  fieldType: TEXT,
+  entity: ROLE_ENTITY,
+  displayName: 'Name',
+  name: 'name',
+  fieldOrder: 1,
```

Comment:
this is the first visible field, so its fieldOrder will be 0, you will also need to change the order in all following fields, make the same change for all of the entities